setup.py

```diff
@@ -24,7 +24,7 @@
 from vsc.install.shared_setup import ag, jt
 
 PACKAGE = {
-    'version': '2.2.0',
+    'version': '2.2.1',
```

Comment:
also bump the minimal vsc-config version

Reply:
fixed :+1: